Traceback (most recent call last):
  File "/Users/guopengpeng/Python/Python_Advanced/pycode1.py", line 66, in <module>
    num = 1/0
ZeroDivisionError: division by zero
Traceback (most recent call last):
  File "/Users/guopengpeng/Python/Python_Advance/pycode1.py", line 66, in <module>
    num = 1/0
ZeroDivisionError: division by zero
